.mergify.yml

```diff
@@ -120,6 +123,7 @@ pull_request_rules:
     conditions:
       # Labels
       - "label!=do not merge"
```

Comment:
Not sure if this will work as separate lines, it says multiple labels should be in a [ ] list....

Reply:
I understand the list in add/remove labels in actions but for conditions?
How to write these:

```
      - "label!=do not merge"
      - "label!=mergify skip"
      - "label=needs: CI"
```

```
      - "label!=['do not merge', mergify skip]"
      - "label=needs: CI"
```
I believe both should have the same result if are loaded from yaml

Reply:
This code snippet was taken from "ready for merge" rule, and that one was working (considering labels), so as it is it should work. Pitty, I can't test here but first merge the config and then on another PR.

Reply:
Ive checked again, also works in case of release label:

```
-label~=^(release-version)
 -label~=^(Release review required)
```

So this will work as it is in the PR